core/static/core/tags/entry.tag

```diff
@@ -0,0 +1,104 @@
+<entry>
+    <div class="col-sm-3">
+        <div class="text-muted small">
+            { project_details.client_details.name }
+        </div>
+        { project_details.name }
+    </div>
+    <div class="col-sm-5 d-flex align-self-end" if={ !edit }>
+        { note }
+    </div>
+    <div class="col-sm-5 d-flex align-self-end" if={ edit }>
+        <input type="text" class="form-control form-control" ref="note" placeholder="Note" value="{ note }">
+    </div>
+    <div class="col-sm-2 d-flex align-self-end" if={ !edit && !run_timer }>
```

Comment:
I don't love the way the logic turned out for this tag (between the Edit and Restart functionality). I tried to change it up a bit but had trouble with the riotjs `if` attribute. Suggestions welcome.